parity/configuration.rs

```diff
@@ -1512,10 +1510,8 @@ mod tests {
 	#[test]
 	fn should_parse_updater_options() {
 		// when
-		let conf0 = parse(&["parity", "--release-track=testing"]);
-		let conf1 = parse(&["parity", "--auto-update", "all", "--no-consensus", "--auto-update-delay", "300"]);
-		let conf2 = parse(&["parity", "--no-download", "--auto-update=all", "--release-track=beta", "--auto-update-delay=300", "--auto-update-check-frequency=100"]);
```

Comment:
If we're removing these, we also need to remove the `assert`s just below in this file. This is leading to tests being broken: https://gitlab.parity.io/parity/parity-ethereum/-/jobs/356325